app/feeds/new/index/template.hbs

```diff
@@ -7,15 +7,15 @@
 </div>
 <div class="add-operator-info">
 	<p>
-		Ready to start adding a feed to Transitland? Step #1 is to find the URL for a <abbr title="General Transit Feed Specification">GTFS</abbr> feed. The feed should live on a transit authority's server or another authoritative source.
+		Ready to add a feed to Transitland? The first step is to specify the web address (URL) to a <abbr title="General Transit Feed Specification">GTFS</abbr> feed. Your link should point directly to a .zip file that is hosted by a transit operator or other authoritative source.
```

Comment:
Is there an alternative to the word "specify"? To me, it feels a little off for the overall feel of transitland, but you think it's the best word to use here, that's ok too.